From c3b6b68d60d24593bf6db62ae9de6d2072ad8e23 Mon Sep 17 00:00:00 2001
From: Andris PE <neandris@gmail.com>
Date: Wed, 27 Mar 2024 21:38:50 +0200
Subject: [PATCH] Reduce default weight.

No need to consume CPU in default case for unrealistic corner case.
loopback invalid thus better dropped at ease.

Signed-off-by: Andris PE <neandris@gmail.com>
---
 root/usr/share/firewall4/templates/ruleset.uc | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/root/usr/share/firewall4/templates/ruleset.uc
+++ b/root/usr/share/firewall4/templates/ruleset.uc
@@ -187,7 +187,7 @@ table inet fw4 {
 	chain prerouting {
 		type filter hook prerouting priority filter; policy accept;
 {% if (fw4.default_option("drop_invalid")): %}
-		iif != "lo" ct state invalid drop comment "!fw4: Drop packets in invalid flow state"
+		ct state invalid drop comment "!fw4: Drop packets in invalid flow state"
 {% endif %}
 {% for (let zone in fw4.zones()): %}
 {%  if (zone.dflags.helper): %}
